### ISSUE TITLE
Dependence between suites?

### ISSUE THREAD

**Issue by ghost** [cylc/cylc-flow#25]
In a parallel trial that feeds off a main operational suite we typically have one or more clock-triggered tasks that trigger when the operational data is expected to be available and then the task implementation has to wait on the data in case it has not been generated yet. This is works perfectly well, but it might be possible to have a new task proxy type that is able to communicate with a remote suite in order to satisfy prerequisites involving remote tasks. This would be more transparent, in terms of suite design, and check-and-wait logic would not be needed in the data retrieval tasks (which would no longer need to be clock triggered either).  This would require the remote suite to remember which of its tasks had completed already (unless the communication was two-way so that a task would not be removed from a suite if a downstream remote task still needed its outputs ... probably not a good idea - it could result in an operational suite being held up by a broken remote suite that feeds off it)
